test/decl/nested/protocol.swift

```diff
@@ -43,7 +43,6 @@ protocol Racoon {
 
 enum OuterEnum {
   protocol C {} // expected-error{{protocol 'C' cannot be nested inside another declaration}}
-  // expected-note@-1{{'C' previously declared here}}
-  case C(C) // expected-error{{invalid redeclaration of 'C'}}
+  case C(C)
```

Comment:
I feel this is a regression, but also feel `invalid redeclaration` is not quite right.
Maybe, it should be `use of undeclared type 'C'`?